hugolib/node_as_page_test.go

```diff
@@ -385,6 +386,25 @@ title = "Deutsche Hugo"
 	assertFileContent(t, expectedFilePath(ugly, "public", "nn", "sect1", "regular2"), true,
 		"Single Title: Page 02")
 
+	// Page Params
+	pages := sites.findAllPagesByKind(KindPage)
+	for _, p := range pages {
+		if p.Path() == "sect1/regular1.en.md" {
```

Comment:
The node_as_page_test was something I built to be able to move from node to page, and is a little bit of everything, and I was thinking of maybe remove it, or move the tests.

Testing TOML to interface{} conversions sounds like a job for a unit test.

As to require: If you have one failing test you have a serious problem -- and fixing that one test is usually enough to fix any tests that come after it. But up to you, my point was: If there are 100 require in a test file, then adding one single assert is at least breaking a pattern.